app/experimenter/experiments/api/v5/types.py

```diff
@@ -37,3 +38,9 @@ class NimbusBucketRangeType(DjangoObjectType):
     class Meta:
         model = NimbusBucketRange
         exclude = ("id", "experiment")
+
+
+class NimbusFeatureConfigType(DjangoObjectType):
+    class Meta:
+        model = NimbusFeatureConfig
+        exclude = ("id",)
```

Comment:
In my current PR, I'm showing ID's for anything that can be updated/deleted. I assume this will need to allow that?

Reply:
Oh yeah that's what I was wondering, k I'll take that out 👍